packages/documents/graphql/schema.js

```diff
@@ -17,6 +17,6 @@ type queries {
     after: String
   ): DocumentConnection!
   # (pre)published document
-  document(slug: String!): Document
+  document(path: String!): Document
```

Comment:
probably would have gone the optional slug and path param route and throwing when both or none are set

Reply:
once we're prd this would be required for smooth deploy. now with on project-r: we can make it a hard breaking change

Reply:
I prefer to keep the backwards compatibility as low as possible during staging phase, to keep it clean in tidy. When we are running in production this going to be different and we'll soon enough have to introduce many such transition solutions.